<?xml version="1.0"?>
<launch>
	<node pkg="user_tracking" type="UserTrackingServer" name="user_tracking" output="screen"/>
</launch>
